client/analysisCtrl.ts

```diff
@@ -15,7 +16,20 @@ import listeners from 'snabbdom/modules/eventlisteners';
 import { Chessground } from 'chessgroundx';
 import { Api } from 'chessgroundx/api';
 import { key2pos, pos2key } from 'chessgroundx/util';
-import { Color, Dests, Pieces, PiecesDiff, Role, Key, Pos, Piece, Variant, Notation, SetPremoveMetadata } from 'chessgroundx/types';
+import {

```

Comment:
Yeah - this happens because i am not using a plain text editor, but an IDE which actually generates those automatically and also hides them most of the time so i am not as bothered by how ugly they look, because 99% of the time all import statements are collapsed. Gotta see if i can set it up somehow to generate less ugly imports